test/e2e/common.ts

```diff
@@ -298,3 +298,25 @@ export async function makeCoordinatorZomeBundle(): Promise<CoordinatorBundle> {
     },
   };
 }
+
+export async function retryUntilTimeout<T>(
+  cb: () => Promise<T>,
+  whileValue: any,
+  timeoutMsg: string,
+  intervalMs: number,
+  timeoutMs: number
+): Promise<T> {
+  const startTime = Date.now();
+  let value = await cb();
+  while (value === whileValue) {
+    const currentTime = Date.now();
+    if (Math.floor(currentTime - startTime) >= timeoutMs)
```

Comment:
`Date.now()` always returns integers, so flooring doesn't do anything.
```suggestion
    if (currentTime - startTime >= timeoutMs)
```

Reply:
Oh, good point. I blind-copied that over from tryorama.

Reply:
https://github.com/holochain/holochain-client-js/pull/353/commits/06a29c32ada14f1fc6a2d5b0502334dde88f43f6